cmd/server/cmd/BUILD

```diff
@@ -13,6 +13,7 @@ go_library(
     ],
     visibility = [
         "//cmd:__subpackages__",
+        "//test/testenv:__subpackages__",
```

Comment:
Sorry didn't mentioned this before. I am not excited with the fact that this library is inside test dir, since it is something that adapter developers will use. @ZackButcher @douglas-reid @mandarjog thoughts ? What should be the right house for the mixer testing framework

Reply:
As we discussed, for now, let's leave it in test dir since it has a dependency on cmd/server/cmd package. Post 0.2, we can refactor the code and move to pkg.

Reply:
I think this is reasonable: `//test` is probably the right place for the test framework to live. Long term we should refactor the test framework to depend on `//pkg/api` and do the wiring up itself, rather than rely on `//cmd/server`'s wiring. I think this will lead to a nicer, more general interface to write tests against; I'd like to see us use essentially this framework to execute our own integration Mixer integration tests down the line.